apps/golden-sample-app/src/styles.scss

```diff
@@ -1,6 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
 
-@import '~@angular/cdk/a11y/a11y';
+@import '../../../node_modules/@angular/cdk/a11y';
```

Comment:
why this change?

Reply:
VSCode is unable to find it's location with the previous import.